### DISCUSSION TITLE
Tag field does not show all values on 1st mouse click in the textbox

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#5011] [Ideas & Feature Requests] [answered]
- Laravel Version: 9
- Nova Version: 4.18.2
- PHP Version: 8.1
- Database Driver & Version: MySQL 8
- Operating System and Version: Windows and Mac
- Browser type and version: Chrome

### Description:
Have a Tag field, click on the textbox, the choices don't appear until you type the 1st letter. Sometimes this is wanted, but usually we like to show all the possible values right away in case we don't remember all the possible choices.

So I suggest we do something like if you want it to be "AJAXy" do ->searchable(), if not, fetch all possible values and show the moment we click on the textbox.

**Comment by sutcliffe121** [comment]
Agree, in my use case, the user cannot always remember which tags are available, so they wish to see a list of available options rather than search.

**Comment by crynobone** [comment, marked as answer]
See https://github.com/laravel/nova-issues/discussions/5077#discussioncomment-5258051
